src/numeric/processes.jl

```diff
@@ -90,7 +90,7 @@ MvNormal(sigma::Float64) = MvNormal(reshape([sigma], 1, 1))
 
 function discretize(mvn::MvNormal)
     n = fill(5, size(mvn.mu))
-    x, w = QE.qnwnorm(n, mvn.mu, mvn.Sigma)
+    x, w = QE.qnwnorm(n, mvn.mu, diagm(vec(mvn.Sigma)))
```

Comment:
and also agree that we should require the yaml file to have a matrix as in
```
exogenous: !Normal
       Sigma: [[sig_a^2 0]; [0 sig_b^2]]
```
and not promote automatically other forms like `Sigma: sig_a^2` or `Sigma: [sig_a^2, sig_b^2]`. They look like they can be intepreted easily (as a 1x1 matrix or a diagonal) but for now they create uncertainty (see bug #33).